src/v/archival/ntp_archiver_service.cc

```diff
@@ -564,7 +568,13 @@ ntp_archiver::upload_tx(upload_candidate candidate) {
 
     cloud_storage::tx_range_manifest manifest(path, tx_range);
 
-    co_return co_await _remote.upload_manifest(_bucket, manifest, fib);
+    // Note: tx-manifest is uploaded using 'remote::upload_manifest'
+    // method but it has the same tags as the segment because it has
+    // the same lifetime as corresponding segment and associated with
+    // the segment.
+    auto tags = cloud_storage::remote::get_segment_tags(_ntp, _rev);
```

Comment:
I wonder if it makes sense to add some `remote::get_tx_manifest_tags()` that just calls `get_segment_tags()`. Might be less error prone for others dealing with `tx-manifest`

Reply:
fixed